TempState.java

```diff
@@ -13,6 +13,8 @@
 	private int cleared;
 	private int nextPiece;
 	private int thisRoundCleared;
+	private int stateOrient;
+	private int stateSlot;
 
```

Comment:
TempState now keeps track of move orientation of peice and which slot it is heading to in order to obtain height of current piece in play and height of current column it is landing in. I could have updated variables stateSlot and stateOrient when makemove is executed but I opted to set it via a setter method afterwards.